### PR TITLE
Fix bigquery

### PR DESCRIPTION
**Pull request by guykeller**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GoogleBigQuery/Integrations/GoogleBigQuery/GoogleBigQuery.yml [hunk 1]
```diff
@@ -11,6 +11,33 @@ configuration:
...
+  name: isFetch
+  required: false
+  type: 8
+- display: Fetch query
+  name: fetch_query
+  required: false
+  type: 0
+- display: Fetch time field
+  additionalinfo: Will be used for determining the time-specifying field of each row. Default is CreationTime.
+  name: fetch_time_field
```
Comment (yuvalbenshalom) [inline Packs/GoogleBigQuery/Integrations/GoogleBigQuery/GoogleBigQuery.yml:38]: please add default

File: Packs/GoogleBigQuery/Integrations/GoogleBigQuery/GoogleBigQuery_test.py [hunk 2]
```diff
@@ -15,3 +18,196 @@ def test_convert_to_string_if_datetime():
     convert_to_string_if_datetime(today)
     test_conversion_for_empty_string = convert_to_string_if_datetime("")
     assert test_conversion_for_empty_string == ""
+
+
+def test_remove_outdated_incident_ids_keep_equal():
+    """
```
Comment (yuvalbenshalom) [inline Packs/GoogleBigQuery/Integrations/GoogleBigQuery/GoogleBigQuery_test.py:24]: please update the case docs. see other tests you added also.

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comments.

File: Packs/GoogleBigQuery/Integrations/GoogleBigQuery/GoogleBigQuery_test.py (hunk 2 above)
Comment (guykeller) [reply Packs/GoogleBigQuery/Integrations/GoogleBigQuery/GoogleBigQuery_test.py:24]: done

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**